core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -279,6 +279,9 @@ public void listStatus() throws Exception {
     FileStatus[] fileStatuses = alluxioHadoopFs.listStatus(path);
     assertFileInfoEqualsFileStatus(fileInfo1, fileStatuses[0]);
     assertFileInfoEqualsFileStatus(fileInfo2, fileStatuses[1]);
+    if (alluxioHadoopFs != null) {
+      alluxioHadoopFs.close();
```

Comment:
you need to mock `mMockFileSystemContext.close()` to prevent null-pointer seen. 
If this is too advanced (seems so), feel free to update and drop this JIRA, and grab another one

@aaudiber this task should not be new contributor